frontend/src/component/project/Project/CreateProject/SelectionButton.tsx

```diff
@@ -262,3 +263,151 @@ type SingleSelectListProps = Pick<
 export const SingleSelectList: FC<SingleSelectListProps> = (props) => {
     return <CombinedSelect {...props} />;
 };
+
+type TableSelectProps = Pick<CombinedSelectProps, 'button' | 'search'> & {
+    onChange: (
+        changeRequestConfig: Record<string, { requiredApprovals: number }>,
+    ) => void;
+    updateProjectChangeRequestConfiguration: {
+        disableChangeRequests: (env: string) => void;
+        enableChangeRequests: (env: string, requiredApprovals: number) => void;
+    };
+    activeEnvironments: {
+        name: string;
+        type: string;
+    }[];
+    projectChangeRequestConfiguration: Record<
+        string,
+        { requiredApprovals: number }
+    >;
+};
+export const TableSelect: FC<TableSelectProps> = ({
+    button,
+    search,
+    onChange,
+    projectChangeRequestConfiguration,
+    updateProjectChangeRequestConfiguration,
+    activeEnvironments,
+}) => {
```

Comment:
This is mostly a duplicate of the general component, so we should probably combine them. For later, though.